base/constants.go

```diff
@@ -117,6 +117,12 @@ const (
 	// Number of kv connections (pipelines) per Couchbase Server node when running in serverless
 	DefaultGocbKvPoolSizeServerless = 1
 
+	// kv connections buffer size per Couchbase Server node
+	DefaultGoCBKvBufferSize = 20 * 1024 * 1024
```

Comment:
If we aren't overriding this value, I don't think we want to specify it at all (i.e. let goCB pick the default value themselves), for compatibility with any future change gocb makes.